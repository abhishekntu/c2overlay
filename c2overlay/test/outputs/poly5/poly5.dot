digraph graphname {
N12 [color="black", label="N9N16_mul_Imm_78_N9_sub_Imm_9504_N16_N12"];
N13 [color="black", label="N4N27_mul_N4_add_N27_N13"];
N10 [color="black", label="mul_N24_N10"];
N11 [color="black", label="N8N29_mul_Imm_288_N8_sub_Imm_5184_N29_N11"];
N16 [color="black", label="N7N14_mul_N7_add_Imm_3456_N14_N16"];
N17 [color="black", label="add_Imm_144_N12_N17"];
N14 [color="black", label="N5N30_mul_N5_add_N30_N14"];
N15 [color="black", label="N6N10_mul_N6_sub_N10_N15"];
N18 [color="black", label="N11N13_mul_N11_add_N13_N18"];
N19 [color="black", label="N17N15_mul_N17_add_Imm_62208_N15_N19"];
N8 [color="black", label="sub_Imm_207_N26_N8"];
N9 [color="black", label="mul_N25_N9"];
N21 [color="black", label="O0_N31_N21"];
N20 [color="black", label="mul_N19_N20"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="sub_Imm_432_N28_N4"];
N5 [color="black", label="sub_Imm_248832_N22_N5"];
N6 [color="black", label="N21N18_mul_Imm_2_N21_sub_N18_N6"];
N7 [color="black", label="N20N23_mul_N20_sub_Imm_2985984_N23_N7"];
N12 -> N18;
N13 -> N9;
N10 -> N13;
N11 -> N20;
N16 -> N15;
N17 -> N6;
N14 -> N13;
N15 -> N10;
N18 -> N5;
N19 -> N7;
N8 -> N16;
N9 -> N21;
N20 -> N18;
N1 -> N20;
N1 -> N15;
N1 -> N10;
N1 -> N14;
N2 -> N13;
N2 -> N9;
N2 -> N17;
N3 -> N7;
N3 -> N12;
N3 -> N19;
N3 -> N6;
N3 -> N11;
N3 -> N8;
N3 -> N16;
N3 -> N18;
N3 -> N4;
N4 -> N19;
N5 -> N14;
N6 -> N15;
N7 -> N14;
}
