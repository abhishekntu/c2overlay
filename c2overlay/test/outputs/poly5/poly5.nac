procedure foo(in s32 x,in s32 y,in s32 z,out s32 D_2448)
{
  localvar s32 D_2447;
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  t1_1 <= mul z,288;
  t1_2 <= mul z,78;
  t2_1 <= ldc 207;
  t2_1 <= sub t2_1,z;
  t2_2 <= sub t1_1,5184;
  t2_3 <= sub t1_2,9504;
  t2_4 <= sub z,432;
  t3_1 <= mul z,t2_1;
  t3_2 <= mul y,t2_2;
  t3_3 <= mul z,t2_3;
  t3_4 <= mul z,t2_4;
  t4_1 <= add x,144;
  t4_2 <= add t3_1,3456;
  t4_3 <= add t3_2,t3_3;
  t4_4 <= add t3_4,62208;
  t5_1 <= mul z,t4_4;
  D_2447 <= mul z,2;
  t6_1 <= sub D_2447,t4_1;
  t6_2 <= sub t4_3,248832;
  t6_3 <= sub t5_1,2985984;
  t7_1 <= mul y,t6_1;
  t7_2 <= mul y,t6_2;
  t8_1 <= sub t7_1,t4_2;
  t8_2 <= add t7_2,t6_3;
  t9_1 <= mul y,t8_1;
  t10_1 <= mul x,t9_1;
  t11_1 <= add t10_1,t8_2;
  D_2448 <= mul x,t11_1;
}
