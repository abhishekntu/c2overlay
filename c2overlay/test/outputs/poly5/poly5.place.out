Netlist file: test.net   Architecture file: test.arch.xml
Array size: 5 x 5 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N1		6	3	0	#0
N2		2	0	0	#1
N3		0	1	0	#2
out:N21		1	0	0	#3
N12_blk		4	4	0	#4
N13_blk		2	1	0	#5
N10_blk		2	2	0	#6
N11_blk		5	2	0	#7
N16_blk		3	3	0	#8
N17_blk		1	2	0	#9
N14_blk		3	2	0	#10
N15_blk		2	3	0	#11
N18_blk		4	3	0	#12
N19_blk		4	1	0	#13
N8_blk		3	4	0	#14
N9_blk		1	1	0	#15
N20_blk		5	3	0	#16
N4_blk		5	1	0	#17
N5_blk		4	2	0	#18
N6_blk		1	3	0	#19
N7_blk		3	1	0	#20
